MAUI/UIKit/GettingStarted.md

```diff
@@ -0,0 +1,73 @@
+---
+layout: post
+title: Getting Started with MAUI UI Kit control | Syncfusion
+description: Learn here about getting started with Syncfusion Essential Studio MAUI UI Kit control, its elements and more.
+platform: maui
+control: MAUI UI Kit
+documentation: ug
+---
+
+# Getting Started with MAUI UI Kit
+
+The UI Kit screens can be added to your application in the following two ways:
+
+1. Using **Essential UI Kit for MAUI** Visual Studio extension.
+
+2. Copying the files from our open-source [GitHub repository]().
+
+
+## Essential UI Kit for MAUI Extension
+
+This is the simplest method to integrate pre-defined screens into your application. The following steps outline how to add screens to your app using our extension:
+
+### Steps to Use Essential UI Kit for MAUI Extension
+
+1. Open Visual Studio.
+
+2. Go to Extension, and then click Manage Extensions as shown in the following screenshot.
+
+   ![Visual Studio Extensions](UI-Kit-images/VS_Extensions.png)
+
+3. Search for **Essential UI Kit for MAUI**, and then install it.
+
+   ![Visual Studio Extensions and Updates](UI-Kit-images/Extension_Update.png)
+
+4. Restart the Visual Studio and allow it to complete the installation. 
+
+5. Now, open an existing MAUI application or create a new application as per your requirements.
+ 
+6. Right-click the MAUI [NETStandard] project, and you can see the **Essential UI Kit for MAUI** option.
```

Comment:
Essential UI Kit for MAUI ---->.   .NET MAUI Essential Ul Kit

Reply:
Changed In Latest Commits